projects/fsastorefrontstyles/scss/overrides/payment-details.scss

```diff
@@ -3,10 +3,18 @@
 
 .cx-msg {
   margin-bottom: 30px;
+  font-size: var(--cx-h3-font-size);
+  font-weight: var(--cx-font-weight-semi);
```

Comment:
This file can be deleted now, right?

Reply:
I thought it is renamed to payment-methods.scss, but somehow it still exists. I'll delete it.